components/occurrence/List.vue

```diff
@@ -90,8 +91,9 @@ const occurrences = computed(() => {
     return filters.filterOccurrences(data.value.occurrences)
 
   const unavailable = data.value.occurrences.filter(o => o.notAvailableAfter !== null)
-  const { visible, hidden } = filters.filterOccurrences(data.value.occurrences.filter(o => o.notAvailableAfter === null))
-  return { visible, hidden: [...hidden, ...unavailable] }
+  const {visible, hidden} = filters.filterOccurrences(data.value.occurrences.filter(o => o.notAvailableAfter === null))
```

Comment:
If possible please format the destruction like the rest of the code. `{visible, hidden}` -> `{ visible, hidden }`